.github/workflows/test-gpu.yml

```diff
@@ -0,0 +1,43 @@
+name: Test GPU
+
+on: [pull_request]
+
+jobs:
+  build-test:
+    runs-on: [self-hosted, gpu]
+    strategy:
+      matrix:
+        include:
+          - itk-git-tag: "c693dbc830859ea10d573301f075e121a4c6c581"
+            cmake-build-type: "MinSizeRel"
+        
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Download ITK
+      run: |
+        pushd ~/ITK
+        git checkout ${{ matrix.itk-git-tag }}
+        popd
+
+    - name: Build ITK
+      run: |
+        pushd ~/ITK-build
+        cmake -DBUILD_SHARED_LIBS:BOOL=ON -DCMAKE_BUILD_TYPE:STRING=${{ matrix.cmake-build-type }} -DBUILD_TESTING:BOOL=OFF -GNinja ../ITK
+        popd
+      shell: bash
+
+    - name: Build ITKVkFFTBackend
+      run: |
+        pwd
+        cmake -DITK_DIR:PATH=/home/ubuntu/ITK-build -DOpenCL_INCLUDE_DIR:PATH=/home/ubuntu/OpenCL-ICD-Loader/inc -DOpenCL_LIBRARY:FILEPATH=/home/ubuntu/OpenCL-ICD-Loader-build/libOpenCL.so.1.2 -DBUILD_TESTING:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=ON -GNinja -S . -B /home/ubuntu/ITKVkFFTBackend-build
```

Comment:
Shouldn't we make all of these paths relative to the home directory, `~`? Would that even work?

Reply:
When building and referencing the paths, we should use paths relative to `${{ github.workspace }}`:

https://docs.github.com/en/actions/learn-github-actions/contexts